resources/leiningen/new/kraken_works/project.clj

```diff
@@ -5,22 +5,27 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [org.clojure/tools.logging "0.3.1"]
-                 [turbovote.resource-config "0.2.0"]
-                 [com.novemberain/langohr "3.6.1"]
-                 [democracyworks/datomic-toolbox "2.0.2"
+                 [turbovote.resource-config "0.2.1"]
+                 [com.novemberain/langohr "3.7.0"]
+                 [democracyworks/datomic-toolbox "2.0.4"
                   :exclusions [com.datomic/datomic-pro]]
-                 [prismatic/schema "1.1.3"]
-                 [com.datomic/datomic-pro "0.9.5394"
+                 [prismatic/schema "1.1.6"]
+                 [com.datomic/datomic-pro "0.9.5561"
                   :exclusions [org.slf4j/slf4j-nop
                                org.slf4j/slf4j-log4j12]]
-                 [com.amazonaws/aws-java-sdk-dynamodb "1.11.6"
+                 [com.amazonaws/aws-java-sdk-dynamodb "1.11.128"
                   :exclusions [commons-codec commons-logging]]
-                 [ch.qos.logback/logback-classic "1.1.7"]
-                 [org.immutant/core "2.1.5"]
-                 [democracyworks/kehaar "0.8.1"]]
-  :plugins [[lein-immutant "2.1.0"]]
+                 [ch.qos.logback/logback-classic "1.2.2"]
+                 [org.immutant/core "2.1.6"]
+                 [democracyworks/kehaar "0.10.4"]]
+  :plugins [[lein-immutant "2.1.0"]
+            [com.carouselapps/jar-copier "0.3.0"]]
```

Comment:
This should be `[com.pupeno/jar-copier "0.4.0"]` now. That fixes the "Coping" bug.